cmd/encryption-v1.go

```diff
@@ -438,6 +608,20 @@ func (o *ObjectInfo) IsEncrypted() bool {
 	return false
 }
 
+func decryptedSize(encryptedSize int64) (int64, error) {
+	if encryptedSize == 0 {
+		return encryptedSize, nil
+	}
+	size := (encryptedSize / (32 + 64*1024)) * (64 * 1024)
+	if mod := encryptedSize % (32 + 64*1024); mod > 0 {
```

Comment:
Please use named constants for these values, so it is easier to understand the meaning of the arithmetic.